timetracker/timetracker/urls.py

```diff
@@ -20,5 +20,5 @@
     path('account/', include('account.urls', namespace='account')),
     path('admin/', admin.site.urls),
     path('vms/', include('vms.urls')),
-    path('', include('core.urls')),
+    path('', include('core.urls'))
```

Comment:
Lists should include commas on the last member as well.
```suggestion
    path('', include('core.urls')),
```